vmm/src/lib.rs

```diff
@@ -457,18 +457,21 @@ impl Vmm {
         Ok(())
     }
 
-    fn vm_info(&self) -> result::Result<VmInfo, VmError> {
+    fn vm_info(&mut self) -> result::Result<VmInfo, VmError> {
```

Comment:
Assuming you don't need `get_balloon_actual()` to be mutable, there's no need to make this function mutable either.